src/ReactCardFlip.jsx

```diff
@@ -53,7 +54,9 @@ class ReactCardFlip extends React.Component {
       },
       flipper: {
         position: 'relative',
-        transformStyle: 'preserve-3d'
+        transformStyle: 'preserve-3d',
+        width: '100%',
```

Comment:
Is there any reason for setting width and height to 100%?

Reply:
I really need this too, would be good to have the option to apply additional styles as I need it taking 100% of my container

Reply:
The reason to set it to 100% was that I can have enclosing div take care of sizing.
Right now styling is a little bit of a hassle.